### ISSUE TITLE
Provide notification that a response to a post has alt text

### ISSUE THREAD

**Issue by ChasBelov** [mastodon/mastodon#32754]
### Pitch

I normally don't boost toots missing alt text. But sometimes I think a toot is sufficiently important that I will reply with a toot supplying an image description.

However, there is no way for a site visitor to know that an image description has been added to the thread. It might be nice to:

- provide a way for a tooter to say that they are adding an image description.
- add an indication on the original post that an image description has been added to the thread.

I'll acknowledge a few arguments against this:

- it can be abused. There is no guarantee that someone checking the box that they have added an image description that they have actually done so.
- it's not fair to people who need alt text to make them read the thread. The image description could be several toots down, placing cognitive load on them as they read comments on the image without knowing what is in the image.
- it normalizes not posting alt text, as tooters count on someone else to rescue them.

### Motivation

I'm not sure this is needed; I wanted to put it out there for discussion. People needing image descriptions would benefit from being able to judge whether it was worth reading a thread for a toot with images missing alt text.
